### PR TITLE
ping360: Fix message frequency filter to a fixed value of 3Hz

### PR DESCRIPTION
**Pull request by patrickelectric**
Fix [#652](https://github.com/bluerobotics/ping-viewer/issues/652)

Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

### CODE REVIEW

**Comment by patrickelectric** [comment]
@Williangalvani can you test via ethernet ?

**Comment by Williangalvani** [comment]
> @Williangalvani can you test via ethernet ?

it is not building...
`../src/sensor/ping360.h:591:42: error: ‘elapsedTime’ was not declared in this scope; did you mean ‘QElapsedTimer’?`

**Comment by patrickelectric** [comment]
@Williangalvani 	my bad, try now.

**Comment by Williangalvani** [comment]
This will still have the value updating every time a new ping arrives, no? Aren't you effetively just changing the cut-off frequency?

I think it you be better if we updated the text every second, with the (pings/time) for that second.
We could also have the maximum and minimum instant frequencies for that second to calculate jitter.

**Comment by Williangalvani** [comment]
Nevermind, just tested here. it looks good enough. it settles in around 3s after changing range and is readable.

What do you think of displaying the jitter, though?

**Comment by jaxxzer** [comment]
> What do you think of displaying the jitter, though?

I don't see any use.

**Comment by patrickelectric** [comment]
@Williangalvani I believe that jitter information can be useful for network transmission. We can improve it in future PRs.

File: src/sensor/ping360.h [hunk 1]
```diff
@@ -576,8 +576,15 @@ class Ping360 : public PingSensor
     struct MessageFrequencyHelper {
         float frequency = 0;
         int lastElapsedTime = 0;
-        // LPF IIR alpha coefficient
-        float alpha = 0.8f;
+
+        /** LPF IIR alpha coefficient
+         *  This creates an order 1 IIR filter with a fixed cutoff frequency.
+         *  The filter equation is descriped as $H(z) = \frac{1-\alpha}{1-\alpha Z}$,
```
Comment (Williangalvani) [inline src/sensor/ping360.h]: "descriped"

**Comment by patrickelectric** [comment]
@Williangalvani 	done.

**Pull request merged**